app/lib/common/dialogs/member_info_drawer.dart

```diff
@@ -174,12 +178,108 @@ class _MemberInfoDrawer extends ConsumerWidget {
   }
 }
 
+class _MemberInfoSkeleton extends StatelessWidget {
```

Comment:
Good to have it on separate skeleton file and folder (If have multiple skeletons).